app/src/main/java/ch/epfl/sdp/MapsActivity.java

```diff
@@ -22,6 +22,10 @@ protected void onCreate(Bundle savedInstanceState) {
         Button mapButton = findViewById(R.id.recenter);
         mapButton.setOnClickListener(v -> mapApi.moveCameraOnCurrentLocation());
 
+        Button scanButton = findViewById(R.id.scanButton);
+        Scan scan = new Scan(new GeoPoint(9.34324, 47.24942), true, 30, mapApi);

```

Comment:
Why do we need to pass the GeoPoint to the Scan class? Should the GeoPoints have already been stored in the Player class?

Reply:
Because each item, before being taken by a player, will appear on the map and therefore will have a location. The GeoPoint stores this location